### PR TITLE
Improve Script Query Runner

### PR DESCRIPTION
**Pull request by ariarijp**
Hi team,

Here is some improvements for Script Query Runner.

I extracted `run_script` function from `run_query` method.
Then, I added some test cases for that.

Thanks,

### CODE REVIEW

File: tests/query_runner/test_script.py [hunk 1]
```diff
@@ -17,3 +18,19 @@ def test_specified(self):
...
+
+    def test_success(self):
+        self.monkeypatch.setattr(subprocess, "check_output", lambda script, shell: "test")
+        self.assertEqual(("test", None), run_script("/foo/bar/baz.sh", True))
+
+    def test_failure(self):
+        self.monkeypatch.setattr(subprocess, "check_output", lambda script, shell: None)
+        self.assertEqual((None, "Error reading output"), run_script("/foo/bar/baz.sh", True))
+        self.monkeypatch.setattr(subprocess, "check_output", lambda script, shell: "")
+        self.assertEqual((None, "Error reading output"), run_script("/foo/bar/baz.sh", True))
```
Comment (arikfr) [inline tests/query_runner/test_script.py]: Maybe return "Empty output from script." message in this (and the `" "`) case?

Comment (ariarijp) [reply tests/query_runner/test_script.py]: Thank you, it makes sense.

File: tests/query_runner/test_script.py [hunk 2]
```diff
@@ -17,3 +18,19 @@ def test_specified(self):
...
+    monkeypatch = MonkeyPatch()
+
+    def test_success(self):
+        self.monkeypatch.setattr(subprocess, "check_output", lambda script, shell: "test")
+        self.assertEqual(("test", None), run_script("/foo/bar/baz.sh", True))
+
+    def test_failure(self):
+        self.monkeypatch.setattr(subprocess, "check_output", lambda script, shell: None)
+        self.assertEqual((None, "Error reading output"), run_script("/foo/bar/baz.sh", True))
+        self.monkeypatch.setattr(subprocess, "check_output", lambda script, shell: "")
```
Comment (ariarijp) [inline tests/query_runner/test_script.py]: Hi, @arikfr 

Could you check it again?

**Comment by arikfr** [comment]
Thanks!

**Pull request merged**